src/frontend/components/VideoWizard/CreateVOD/UploadVideoForm/index.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+
+import {
+  UploadManagerStatus,
+  useUploadManager,
+} from 'components/UploadManager';
+import { useCurrentVideo } from 'data/stores/useCurrentRessource/useCurrentVideo';
+import { uploadState } from 'types/tracks';
+import { Nullable } from 'utils/types';
+
+import { UploadVideoDropzone } from './UploadVideoDropzone';
+import { UploadVideoPreview } from './UploadVideoPreview';
+import { UploadVideoProgress } from './UploadVideoProgress';
+import { UploadVideoRetry } from './UploadVideoRetry';
+
+interface UploadVideoFormProps {
+  onRetry: () => void;
+  setVideoFile: (videoFile: Nullable<File>) => void;
+}
+
+export const UploadVideoForm = ({
+  onRetry,
+  setVideoFile,
+}: UploadVideoFormProps) => {
+  const video = useCurrentVideo();
+  const { resetUpload, uploadManagerState } = useUploadManager();
+  const [src, setSrc] = useState<Nullable<string>>(null);
+
+  if (!src) {
+    return (
+      <UploadVideoDropzone
+        setVideoFile={(videoFile) => {
+          setVideoFile(videoFile);
+          setSrc(URL.createObjectURL(videoFile));
+        }}
+      />
+    );
+  }
+
+  if (uploadManagerState[video.id]) {
```

Comment:
case when the video is complete seems not managed. We stay with the progress bar at 100% and we are never redirected to the dasboard